tests/insert_test.go

```diff
@@ -26,25 +26,24 @@ func TestInsert(t *testing.T) {
 	t.Run("Regression test for #1573 without actual pipes", func(t *testing.T) {
```

Comment:
This is not really a good test for #1573 now that I think about it but I'm not sure how to test stuff that needs piping bytes into Gopass

Reply:
Good question, I don't know.